pennylane/qaoa/cost.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""
+This file contains functions that generate QAOA cost Hamiltonians corresponding to
+different optimization problems.
+"""
+
+import networkx as nx
+import pennylane as qml
+
+
+def MaxCut(graph):
+    r"""A method that builds a QAOA cost Hamiltonian corresponding to the MaxCut problem, for a given graph.
+
+    The goal of the MaxCut problem for a particular graph is to find a partition of nodes into two sets, such
+    that the number of edges in the graph with endpoints in different sets is maximized. Formally, we wish to
+    find the cut of the graph such that the number of edges crossing the cut is maximized
+    (see `Cut (graph theory) <https://en.wikipedia.org/wiki/Cut_(graph_theory)>`__).
+
+    Recommended mixer Hamiltonian: ~.qaoa.x_mixer
+
+    Recommended initialization circuit: ~.templates.even_superposition
```

Comment:
I don't believe this template exists?

Reply:
You're right, it was something we were maybe planning on adding, but I'll take it out for now.